config/environment.js

```diff
@@ -2,6 +2,11 @@
 
 module.exports = function(environment) {
   var ENV = {
+    OSF: {
```

Comment:
We'll want to find a way to configure this we don't need to include credentials in our enviornment.js file. 

Probably something akin to how we do default/local settings on OSF would work, but there may be an existing convention in the Ember community.